src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
```

Comment:
:beetle: You must put an entry in the constructor initialization list to ensure this _pointer_ is nulled at start-up otherwise it could be a random value and then you will get a segmentation fault pretty quickly when the code after the `if(...) {...}` block starts to use the garbage value, i.e. ensure you put a line:

    , mpAboutDlg(nullptr) // or 0 like the existing things use..!

in the right place (in the same relative position as it is in the `.h` file to all the other members of the `mudlet` class) in the list of items before the opening `{` for the `mudlet::mudlet()` definition in this file.